engine/swarm/stack-deploy.md

```diff
@@ -0,0 +1,242 @@
+---
+description: How to deploy a stack to a swarm
+keywords:
+- guide, swarm mode, composefile, stack, compose, deploy
+title: Deploy a stack to a swarm
+---
+
+When running Docker Engine in swarm mode, you can use `docker stack deploy` to
+deploy a complete application stack to the swarm. The `deploy` command accepts
+a stack description in the form of a [Compose file](/compose/compose-file.md).
+
+The `docker stack deploy` command supports any Compose file of version "3.0" or
+above. If you have an older version, upgrading is simple - see the
```

Comment:
Let's not editorialize about how simple it is. :) Something like "If you have an older version, see <link>."